composer/src/main/kotlin/com/gojuno/composer/Consts.kt

```diff
@@ -0,0 +1,5 @@
+package com.gojuno.composer
+
+val EXTERNAL_STORAGE = "/storage/emulated/0"
```

Comment:
I meant retrieving the actual value from environment on device using something like `adb shell printenv EXTERNAL_STORAGE`.
AFAIR it is mandatory: https://source.android.com/devices/storage/config-example

Reply:
Do you need me to try and do this or will you add something more generic to `commander` ?

Reply:
I'll add it to commander now